packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1313,21 +1331,47 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setUpLocusInfoSelfListener() {
-    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_MUTED_BY_OTHERS, (payload) => {
-      if (payload) {
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_LOCAL_UNMUTE_REQUIRED, () => {
+      if (this.audio) {
+        this.audio.handleServerLocalUnmuteRequired();
         Trigger.trigger(
           this,
           {
             file: 'meeting/index',
             function: 'setUpLocusInfoSelfListener'
           },
-          EVENT_TRIGGERS.MEETING_SELF_MUTED_BY_OTHERS,
+          EVENT_TRIGGERS.MEETING_SELF_UNMUTED_BY_OTHERS,
+          {}
+        );
+      }
+    });
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_REMOTE_MUTE_STATUS_UPDATED, (payload) => {
+      if (payload) {
+        if (this.audio) {
+          this.audio.handleServerRemoteMuteUpdate(payload.muted, payload.unmuteAllowed);
+        }
+        Trigger.trigger(
+          this,
           {
-            payload
-          }
+            file: 'meeting/index',
+            function: 'setUpLocusInfoSelfListener'
+          },
+          payload.muted ? EVENT_TRIGGERS.MEETING_SELF_MUTED_BY_OTHERS : EVENT_TRIGGERS.MEETING_SELF_UNMUTED_BY_OTHERS,
+          payload.muted ? {unmuteAllowed: payload.unmuteAllowed} : {}
         );
       }
     });
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_REQUEST_TO_UNMUTE_RECEIVED, () => {
```

Comment:
I don't know what better names to give it. For me the "SELF_" at the beginning is confusing, so I can remove it so it would be:
REQUEST_TO_UNMUTE_RECEIVED - means that host requested you to unmute yourself
LOCAL_UNMUTE_REQUIRED - means that host requires you to unmute yourself (this is a hard requirement so it's fulfilled by the SDK, we don't give the user or developer a choice). Maybe to make it more clear, we could name it FORCED_LOCAL_UNMUTE? but this is a locus module event and at that point the unmute is not done yet, it's done by the meeting object, so that's why "required" fits better.

Maybe just to make it more clear we should align them more, and add comments, so they would be:
LOCAL_UNMUTE_REQUESTED - it's just a request, user has a choice
LOCAL_UNMUTE_REQUIRED - it's a requirement, user has no choice (SDK does the unmute)